### DISCUSSION TITLE
Unit tests

### DISCUSSION THREAD

**Discussion by aguaviva** [v923z/micropython-ulab#614] [General]
I'd be great to have some sort of scheme to create and provide unit tests for this great lib. Ideally, to make your life easier, any bug that is filed should come with a unit tests that reproduces the issue. 

Thanks

**Comment by v923z** [comment]
There are test scripts that run via the CI workflow: https://github.com/v923z/micropython-ulab/tree/master/tests. Is that what you have in mind?
